common/app/templates/inlineJS/blocking/applyRenderConditions.scala.js

```diff
@@ -101,5 +140,10 @@
     if (baseFontSize && parseInt(baseFontSize, 10) !== 16) {
         documentElement.style.fontSize = baseFontSize
     }
+
+    if (hideReaderRevenue()) {
+        docClass += ' hide-reader-revenue';
```

Comment:
It does seem worthwhile having the fine-grained control just as we do in JS. I just wonder whether we'll have to do this all over the place:

```
  .is-recent-contributor &,
  .is-paying-member & {
    ...
  }
```

Maybe that's OK